include/bluetooth/services/fast_pair.h

```diff
@@ -77,13 +80,35 @@ enum bt_fast_pair_battery_comp {
 	BT_FAST_PAIR_BATTERY_COMP_COUNT
 };
 
+/** @brief Fast Pair discoverable advertising info. Currently empty. */
+struct bt_fast_pair_disc_adv_info {};
+
+/** @brief Fast Pair not discoverable advertising info. */
+struct bt_fast_pair_not_disc_adv_info {
+	/** Advertising type. According to Fast Pair specification, when no Account Key has been
+	 *  saved on the device, this field has no effect on the advertising data.
+	 */
+	enum bt_fast_pair_not_disc_adv_type type;
+
+	/** Fast Pair advertising battery mode. */
+	enum bt_fast_pair_adv_battery_mode battery_mode;
+};
+
 /** @brief Fast Pair advertising config. Used to generate advertising packet. */
 struct bt_fast_pair_adv_config {
 	/** Fast Pair advertising mode. */
-	enum bt_fast_pair_adv_mode adv_mode;
+	enum bt_fast_pair_adv_mode mode;
 
-	/** Fast Pair advertising battery mode. */
-	enum bt_fast_pair_adv_battery_mode adv_battery_mode;
+	/** Fast Pair advertising info. Content depends on Fast Pair advertising mode
+	 *  (see @ref mode field).
+	 */
+	union {
+		/** Fast Pair discoverable advertising info. */
```

Comment:
Maybe we could explicitly mention the related enums here (right above each struct)?